synapse/models/infotech.py

```diff
@@ -1636,6 +1647,25 @@ def getModelDefs(self):
                     ('author:contact', ('ps:contact', {}), {
                         'doc': 'The contact information for the author of the ATT&CK Flow diagram.'}),
                 )),
+                ('it:mitre:attack:datasource', {}, (
+                    ('name', ('str', {'lower': True, 'onespace': True}), {
+                        'doc': 'The name of the datasource.'}),
+                    ('description', ('str', {}), {
```

Comment:
Are the descriptions long enough that we should add `'disp': {'hint': 'text'},` on them?